src/plugins/lanplus/lanplus.c

```diff
@@ -897,58 +904,57 @@ ipmi_lan_poll_recv(struct ipmi_intf * intf)
+ * Session Response payload starts
  *
  * returns 0 on success, 1 on error
  */
-void
-read_open_session_response(struct ipmi_rs * rsp, int offset)
+void read_open_session_response(struct ipmi_rs *rsp, int offset)
 {
 	memset(&rsp->payload.open_session_response, 0,
-			 sizeof(rsp->payload.open_session_response));
+	       sizeof(rsp->payload.open_session_response));
 
-	 /*  Message tag */
-	 rsp->payload.open_session_response.message_tag = rsp->data[offset];
+	/*  Message tag */
+	rsp->payload.open_session_response.message_tag = rsp->data[offset];
 
-	 /* RAKP response code */
-	 rsp->payload.open_session_response.rakp_return_code = rsp->data[offset + 1];
+	/* RAKP response code */
+	rsp->payload.open_session_response.rakp_return_code =
+		rsp->data[offset + 1];
 
-	 /* Maximum privilege level */
-	 rsp->payload.open_session_response.max_priv_level = rsp->data[offset + 2];
+	/* Maximum privilege level */
+	rsp->payload.open_session_response.max_priv_level =
+		rsp->data[offset + 2];
 
-	 /*** offset + 3 is reserved ***/
+	/*** offset + 3 is reserved ***/
 
-	 /* Remote console session ID */
-	 memcpy(&(rsp->payload.open_session_response.console_id),
-			rsp->data + offset + 4,
-			4);
-	 #if WORDS_BIGENDIAN
-	 rsp->payload.open_session_response.console_id =
-		 BSWAP_32(rsp->payload.open_session_response.console_id);
-	 #endif
+	/* Remote console session ID */
+	memcpy(&(rsp->payload.open_session_response.console_id),
+	       rsp->data + offset + 4, 4);
+#if WORDS_BIGENDIAN
+	rsp->payload.open_session_response.console_id =
+		BSWAP_32(rsp->payload.open_session_response.console_id);
+#endif
 
 	/* only tag, status, privlvl, and console id are returned if error */
-	 if (rsp->payload.open_session_response.rakp_return_code !=
-		  IPMI_RAKP_STATUS_NO_ERRORS)
-		 return;
-
-	 /* BMC session ID */
-	 memcpy(&(rsp->payload.open_session_response.bmc_id),
-			rsp->data + offset + 8,
-			4);
-	 #if WORDS_BIGENDIAN
-	 rsp->payload.open_session_response.bmc_id =
-		 BSWAP_32(rsp->payload.open_session_response.bmc_id);
-	 #endif
-
-	 /* And of course, our negotiated algorithms */
-	 rsp->payload.open_session_response.auth_alg      = rsp->data[offset + 16];
-	 rsp->payload.open_session_response.integrity_alg = rsp->data[offset + 24];
-	 rsp->payload.open_session_response.crypt_alg     = rsp->data[offset + 32];
-}
+	if (rsp->payload.open_session_response.rakp_return_code
+	    != IPMI_RAKP_STATUS_NO_ERRORS)
+		return;
 
+	/* BMC session ID */
+	memcpy(&(rsp->payload.open_session_response.bmc_id),
+	       rsp->data + offset + 8, 4);
+#if WORDS_BIGENDIAN
+	rsp->payload.open_session_response.bmc_id =
+		BSWAP_32(rsp->payload.open_session_response.bmc_id);
+#endif
+
+	/* And of course, our negotiated algorithms */
+	rsp->payload.open_session_response.auth_alg = rsp->data[offset + 16];
+	rsp->payload.open_session_response.integrity_alg =
+		rsp->data[offset + 24];
+	rsp->payload.open_session_response.crypt_alg = rsp->data[offset + 32];
```

Comment:
Need to do something with these magic offsets. As well as with the `8` and `4` above.
Besides, the old formatting was much better readable. I prefer violating the line length over making the code less readable.